HeterogeneousCore/CUDAUtilities/test/host_noncached_unique_ptr_t.cpp

```diff
@@ -0,0 +1,22 @@
+#include "catch.hpp"
+
+#include "HeterogeneousCore/CUDAUtilities/interface/host_noncached_unique_ptr.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/requireCUDADevices.h"
+
+TEST_CASE("host_noncached_unique_ptr", "[cudaMemTools]") {
+  requireCUDADevices();
```

Comment:
The others seem to use 
```
if(not hasCUDADevices()) {
  return;
}
```
Why is this one different?

Reply:
I missed it, it should be as the others.